x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -32539,14 +32539,10 @@
     "xpack.observability.obltNav.apm.syntheticsGroupTitle": "Synthetics",
     "xpack.observability.obltNav.applications": "Applications",
     "xpack.observability.obltNav.devTools": "Outils de développeur",
-    "xpack.observability.obltNav.getStarted": "Démarrer",
```

Comment:
This was renamed to `Add data`, so I added a new key `xpack.observability.obltNav.addData`